dot/network/host.go

```diff
@@ -179,7 +179,9 @@ func (h *host) send(p peer.ID, msg Message) (err error) {
 		return err
 	}
 
-	// TODO: append leb128 variable-length encoding #484
+	lenBytes := uint64ToLEB128(uint64(len(encMsg)))
+	encMsg = append(lenBytes, encMsg...)
```

Comment:
maybe break this up and use `lebEnc` to be clear what we are appending the message to:

```
lenBytes := uint64(len(encMsg))
lebEnc := uint64ToLEB128(lenBytes)
encMsg = append(lebEnc, encMsg...)
```

Reply:
done